base/changes.txt

```diff
@@ -10,6 +10,12 @@ not part of the distribution.
 All changes above are only part of the development branch for the next release.
 ================================================================================
 
+2025-06-05  Frank Mittelbach  <Frank.Mittelbach@latex-project.org>
+
+	* ltexpl.dtx (section{Document-level command names for \pkg{expl3} functions}):
+	Provide T and F variants of \IfExplAtLeastTF with a
+	correct implementation} (gh/1752)
```

Comment:
A surplus `}` was sneaked into `changes.txt`, finally the [announcement text](https://ctan.org/ctan-ann/id/64d0e88376846e77@hogwart) of LaTeX 2025-06-01 PL1.